home/ivp-planning/advanced-tools/path-tool.md

```diff
@@ -82,7 +82,7 @@ The "Area" option also affects the path in 3D-panel so that it can be used to hi
 
 ## Path renaming:
 
-Paths can be renamed at any time via the [path panel](../user-interface/path-panel.md). After **right clicking on the desired path** and choosing **"Rename"**, the name of the path will get a blue square as background. It can now be changed by **entering the new name**, followed by the **Enter**.
+Paths can be renamed at any time via the [hierarchy](../user-interface/the-machine-list.md). After **right clicking on the desired path** and choosing **"Rename"**, the name of the path will get a blue square as background. It can now be changed by **entering the new name**, followed by the **Enter**.
```

Comment:
In fact this is only one possible option to rename things. You can also rename by pressing F2 or by clicking the pen icon in the properties panel. Is there already a renaming section for machines that we could link to?

Reply:
changed it to "Paths can be [renamed](/the-tree-view/renaming-objects-and-folders) at any time just like any other object."

-> again not done before because of general rework